explore/realspace.py

```diff
@@ -38,64 +38,68 @@ def Rx(angle):
     R = [[1, 0, 0],
          [0, +cos(a), -sin(a)],
          [0, +sin(a), +cos(a)]]
-    return np.matrix(R)
+    return np.array(R)
 
 def Ry(angle):
     """Construct a matrix to rotate points about *y* by *angle* degrees."""
     a = radians(angle)
     R = [[+cos(a), 0, +sin(a)],
          [0, 1, 0],
          [-sin(a), 0, +cos(a)]]
-    return np.matrix(R)
+    return np.array(R)
 
 def Rz(angle):
     """Construct a matrix to rotate points about *z* by *angle* degrees."""
     a = radians(angle)
     R = [[+cos(a), -sin(a), 0],
          [+sin(a), +cos(a), 0],
          [0, 0, 1]]
-    return np.matrix(R)
+    return np.array(R)
 
 def pol2rec(r, theta, phi):
     """
     Convert from 3D polar coordinates to rectangular coordinates.
     """
     theta, phi = radians(theta), radians(phi)
     x = +r * sin(theta) * cos(phi)
-    y = +r * sin(theta)*sin(phi) 
-    z = +r * cos(theta) 
+    y = +r * sin(theta) * sin(phi)
+    z = +r * cos(theta)
     return x, y, z
 
 def rotation(theta, phi, psi):
-    """
+    r"""
     Apply the jitter transform to a set of points.
     Points are stored in a 3 x n numpy matrix, not a numpy array or tuple.
+    View is in degrees using nautical angles with roll $\psi$ around $c$,
+    pitch $\theta$ around $b$ and yaw $\phi$ around $a$.
     """
-    return Rx(phi)*Ry(theta)*Rz(psi)
+    return Rx(phi) @ Ry(theta) @ Rz(psi)
 
 def apply_view(points, view):
-    """
-    Apply the view transform (theta, phi, psi) to a set of points.
+    r"""
+    Apply the view transform $(\theta, \phi, \psi)$ to a set of points.
     Points are stored in a 3 x n numpy array.
-    View angles are in degrees.
+    View is in degrees using $z$-$y$-$z$ Euler angles $\phi$-$\theta$-$\psi$.
     """
     theta, phi, psi = view
-    return np.asarray((Rz(phi)*Ry(theta)*Rz(psi))*np.matrix(points.T)).T
-
+    R = Rz(phi) @ Ry(theta) @ Rz(psi)
+    return (R @ points.T).T
 
 def invert_view(qx, qy, view):
-    """
-    Return (qa, qb, qc) for the (theta, phi, psi) view angle at detector
-    pixel (qx, qy).
-    View angles are in degrees.
+    r"""
+    Return $(q_a, q_b, q_c)$ for the $(\theta, \phi, \psi)$ view angle at
+    detector pixel corresponding to $(q_x, q_y)$.
+    View is in degrees using $z$-$y$-$z$ Euler angles $\phi$-$\theta$-$\psi$.
```

Comment:
To make it maybe clearer that it is not a mispelling:
using a $z$-$y$-$z$ rotation sequence of the Euler angles ...